client/blocks/support-article-dialog/dialog.jsx

```diff
@@ -60,28 +66,31 @@ export const SupportArticleDialog = ( {
 		}
 	}, [ actionUrl, post ] );
 
+	const handleCloseDialog = () => {
+		closeSupportArticleDialog();
+		updateSupportArticle( null );
+	};
+
 	return (
 		<Dialog
 			isVisible
 			additionalClassNames="support-article-dialog"
 			baseClassName="support-article-dialog__base dialog"
 			buttons={ [
-				<Button onClick={ closeSupportArticleDialog }>
-					{ translate( 'Close', { textOnly: true } ) }
-				</Button>,
+				<Button onClick={ handleCloseDialog }>{ translate( 'Close', { textOnly: true } ) }</Button>,
 				actionUrl && (
 					<Button
 						href={ actionUrl }
 						target={ actionIsExternal ? '_blank' : undefined }
 						primary
-						onClick={ () => ( actionIsExternal ? noop() : closeSupportArticleDialog() ) }
+						onClick={ () => ( actionIsExternal ? noop() : handleCloseDialog() ) }
```

Comment:
I know this was here before, but do we really need to declare another function that invokes a function? 

In other words, this can be simplified to:

```suggestion
						onClick={ actionIsExternal ? noop : handleCloseDialog }
```